openmrs/apps/registration/fieldValidation.js

```diff
@@ -16,5 +16,19 @@ Bahmni.Registration.customValidator = {
             return value.match(/^\w+$/);
         },
         errorMessage: "REGISTRATION_CASTE_TEXT_ERROR_KEY"
+    },
+    "FATHER_NAME" : {
+        method: function (name, value, personAttributeDetails) {
+            var regexCharacters = '^[a-záàãâéèêẽíìóòõôúùçA-ZÁÀÃÂÉÈÊẼÍÌÓÒÔÕÚÙÇ ]+$';
```

Comment:
Can we move the regex to the field validation json object in app.json file, same as the other fields?

Reply:
@nthfloor i will remove Father and Mother name pattern in app.json file